otelcolbuilder/Makefile

```diff
@@ -83,7 +83,7 @@ _builder:
 .PHONY: _gobuild
 _gobuild:
 	(cd cmd && \
-		CGO_ENABLED=$(CGO_ENABLED) go build -v \
+		GODEBUG=netdns=go CGO_ENABLED=$(CGO_ENABLED) go build -v \
```

Comment:
I'd prefer if we had the `GODEBUG` variable defined at the top of the Makefile, and add a comment there explaining why we set this.